tests/integration/conftest.py

```diff
@@ -365,35 +319,26 @@ def freeze_time_fixture() -> str:
     return "2022-01-01 15:00:00"
 
 
-@pytest_asyncio.fixture(scope="function", name="restart_time_range_app")
-async def restart_time_range_app_fixture(application: Application):
+@pytest_asyncio.fixture(scope="function", name="prepare_restart_time_range")
+async def prepare_restart_time_range_fixture(application: Application):
     """Application with restart-time-range configured."""
     await application.set_config({"restart-time-range": "03-05"})
 
-    yield application
+    yield
```

Comment:
Should this maybe yield the value of the config that was set? It probably won't be used, just seems like a fixture should yield something

Reply:
Yielded application and renamed this to `app_with_restart_config` as a fixture, thank you!